apps/testing/lib/source_websocket.ex

```diff
@@ -0,0 +1,3 @@
+defmodule SourceWebsocket do
+  use Glock.Socket
```

Comment:
> Glock.Socket

You're making this up...

Reply:
You're right, i did make it up. When i wrote the library: https://hexdocs.pm/glock